src/game/server/swarm/asw_player.h

```diff
@@ -91,6 +91,7 @@ class CASW_Player : public CBaseMultiplayerPlayer, public IASWPlayerAnimStateHel
 	CASW_Inhabitable_NPC *GetSpectatingNPC() const;
 	HSCRIPT ScriptGetSpectatingNPC() const;
 	CNetworkHandle( CASW_Inhabitable_NPC, m_hSpectating );
+	CNetworkVar( int, m_iSpectatorIndexes );
```

Comment:
should be unsigned if we're allowing a 32nd player

Reply:
You can replace `int` with `unsigned` and it should work fine.

Reply:
hm also it should work either way, 31st player will be -2147483648 instead of 2147483648 but bitwise operations will still work correctly (32nd player is the one becoming 1)